include/crm/common/xml_names.h

```diff
@@ -14,15 +14,19 @@
 extern "C" {
 #endif
 
-/* This file defines constants for XML element and attribute names.
- *
- * For consistency, new constants should start with "PCMK_", followed by:
- * * "XE" for XML element names
- * * "XA" for XML attribute names
- * * "OPT" for cluster option (property) names
- * * "META" for meta-attribute names
- * * "VALUE" for enumerated values (such as for options or for XML attributes)
- * * "NODE_ATTR" for node attribute names
+/**
```

Comment:
If desired, this can be `/*!` for consistency with most of our doxygen comments. Interestingly a lot (but not all) of our `\file` directives are enclosed by these Javadoc-style double-asterisk comment blocks.

Reply:
might as well -- I don't remember how the difference got established